models/activities.js

```diff
@@ -179,7 +179,7 @@ if (Meteor.isServer) {
       watchers = _.union(watchers, card.watchers || []);
       params.card = card.title;
       title = 'act-withCardTitle';
-      params.url = card.absoluteUrl();
+      params.url = card.originRelativeUrl();
```

Comment:
Should we use `absoluteUrl()` here instead ?

Reply:
@Majed6 

Please try to get it working everywhere with relative URLs.